tgui/packages/tgui/interfaces/PreferencesMenu/useRandomToggleState.ts

```diff
@@ -0,0 +1,6 @@
+import { useLocalState } from "../../backend";
+
+export const useRandomToggleState: (context) => [
+  boolean, (newValue: boolean) => void
+]
+  = context => useLocalState(context, "randomToggle", false);
```

Comment:
Is it not possible to just specify `useLocalState<boolean>` and get those types inferred?

Reply:
It *can*, I'm just always a fan of explicitly contracting function arguments/return types.

Reply:
There is little point in doing that, as it would just be another point of breakage under refactoring.